Controller/RegistrationController.php

```diff
@@ -96,7 +96,7 @@ public function registerAction(Request $request)
     /**
      * Tell the user to check his email provider.
      */
-    public function checkEmailAction()
+    public function checkEmailAction(Request $request)
```

Comment:
It should, but it doesn't need the same signature: https://3v4l.org/ULUpv 

cc @greg0ire